File: InSearchOfAnEasyProblem.cpp
#include <iostream>
using namespace std;

int main(){
  int number;
  bool hard=false;

  cin>>number;

  int opinions[number];
  for(int i=0; i<number; i++){
    cin>>opinions[i];

    if(opinions[i] == 1){
      hard = true;
      cout<<"HARD"<<endl;
      break;
    }
  }

  if(hard != true){
    cout<<"EASY"<<endl;
  }

  return 0;
}

File: InSearchOfAnEasyProblem.py
number = raw_input();
myArray = raw_input();

# for i in range(int(number)):
#     value = raw_input();
#
#     myArray.append(int(value))
#
# if 1 in myArray:
#     print "HARD"
# else:
#     print "EASY"
myArray = myArray.split()


for x in myArray:
    x = int(x)

if 1 in myArray:
    print "HARD"
else:
    print "EASY"
